main/settings_celery.py

```diff
@@ -148,6 +148,12 @@
         "task": "learning_resources_search.tasks.update_featured_rank",
         "schedule": crontab(minute=30, hour=7),  # 3:30am EST
     },
+    "scrape-marketing-pages-every-1-hour": {
+        "task": "vector_search.tasks.scrape_marketing_pages",
```

Comment:
Every hour seems like a lot, I doubt the marketing pages will change that often.  The main sloan task only runs once per day.